Microsoft.Identity.Web/AuthorityHelpers.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Identity.Web
+{
+    public static class AuthorityHelpers
+    {
+        internal static bool IsV2Authority(string authority)
```

Comment:
Added this helper to check if the authority has `v2.0` already, before appending it to it on `AddSignIn` method

Reply:
did you include unit tests for this? also, not a fan of having "v2" in the wording, as things change so much. is there way to make it more adapt for change? @jmprieur 

---
In reply to: [371532491](https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2/pull/272#discussion_r371532491) [](ancestors = 371532491)